### PR TITLE
python.pkgs.ptyprocess: 0.5 -> 0.5.2 

### PR DESCRIPTION
**Pull request by FRidh**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/ptyprocess/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "ptyprocess";
+  version = "0.5.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256= "e64193f0047ad603b71f202332ab5527c5e52aa7c8b609704fc28c0dc20c4365";
```
Comment (dotlambda) [inline pkgs/development/python-modules/ptyprocess/default.nix]: Missing blank.
And why don't you use short hash? `0ra31k10v3629xq0kdn8lwmfbi97anmk48r03yvh7mks0kq96hg6`

Comment (FRidh) [reply pkgs/development/python-modules/ptyprocess/default.nix]: Simple copy-paste from PyPI ;-)

**Comment by FRidh** [comment]
Pushed as 356b5532392a0d6df2c99f28964803d989e31f89

File: pkgs/development/python-modules/ptyprocess/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/ptyprocess/default.nix]: I must be blind. Where did you find sha256? I only see md5

Comment (FRidh) [reply pkgs/development/python-modules/ptyprocess/default.nix]: https://pypi.org/project/ptyprocess/#files

**Pull request closed without merging**